digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala

```diff
@@ -269,6 +269,12 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
     getByMunicipality(municipalityCode, mapRoadLinks, roadLinks, changeInfo, floatingAdjustment(adjustmentOperation, createOperation))
   }
 
+  def getByMunicipality(municipality: Int, administrativeClass: AdministrativeClass): Seq[PersistedAsset] = {
+    val (roadLinks, changeInfo) = roadLinkService.getRoadLinksWithComplementaryAndChangesFromVVH(municipality)
+    val mapRoadLinks = roadLinks.filterNot(_.administrativeClass == administrativeClass).map(l => l.linkId -> l).toMap
+    getByMunicipality(municipality, mapRoadLinks, roadLinks, changeInfo, floatingAdjustment(adjustmentOperation, createOperation))
```

Comment:
On Pedestrian Crossing visualization  I found an issue here. This method set the trafficSigns to floating

Reply:
def getByMunicipality(municipalityCode: Int, filterAdministrativeClass: AdministrativeClass): Seq[PersistedAsset] = {
    val (roadLinks, changeInfo) = roadLinkService.getRoadLinksWithComplementaryAndChangesFromVVH(municipality)	    val (roadLinks, changeInfo) = roadLinkService.getRoadLinksWithComplementaryAndChangesFromVVH(municipalityCode)
    val mapRoadLinks = roadLinks.filterNot(_.administrativeClass == administrativeClass).map(l => l.linkId -> l).toMap	    val mapRoadLinks = roadLinks.map(l => l.linkId -> l).toMap
    getByMunicipality(municipality, mapRoadLinks, roadLinks, changeInfo, floatingAdjustment(adjustmentOperation, createOperation))	    val assets = getByMunicipality(municipalityCode, mapRoadLinks, roadLinks, changeInfo, floatingAdjustment(adjustmentOperation, createOperation))
     val filterRoadLinks = mapRoadLinks.filterNot(_._2.administrativeClass == filterAdministrativeClass)
    assets.filter{a => filterRoadLinks.get(a.linkId).nonEmpty}
  }